app/views/catalog/_show_sections/_default_contents.html.erb

```diff
@@ -13,14 +12,7 @@
         <span class="label">Type</span>
         <%= content_ds.ng_xml.root['type'] %>
         <ul>
-          <% apo_pid = nil
-          if admin_policy_object
-            apo_pid = admin_policy_object.pid
-          end
-          
-          can_manage = current_user.is_admin || current_user.is_manager || (apo_pid && object.can_manage_content?(current_user.roles(apo_pid)))
-          can_view = current_user.is_admin || current_user.is_manager || (apo_pid && object.can_view_content?(current_user.roles(apo_pid)))
```

Comment:
Notice here that `is_manager` is included.  This is part of the clarification of how `is_admin` differs from `is_manager`.